package.json

```diff
@@ -32,7 +33,7 @@
   },
   "engines": {
     "npm": ">=7.0.0",
-    "node": "16"
+    "node": ">=16.9"
```

Comment:
btw we set a fixed version here to just avoid any surprises, because its a complex product, not a library, I would rather keep it

Reply:
to be clear: I am against this change, can create additional work right now